src/components/atoms/button/index.jsx

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Button } from 'react-native';
+
+export default function ButtonComponent({ onPress, title, color }) {
+  return <Button onPress={onPress} title={title} color={color} />;
+}
+
+Button.defaultProps = {
+  color: null,
+  title: null,
+  onPress: () => {}
```

Comment:
Might it be the case that these are all `required`? 

`color` could be the only one worth a `default` here 🤔

Reply:
@jh3y good idea!